src/views/open-api-docs-preview/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// Third-party
+import { useState } from 'react'
+// View
+import OpenApiDocsView from 'views/open-api-docs-view'
+// Components
+import SidebarLayout from 'layouts/sidebar-layout'
+import { OpenApiPreviewInputs } from 'views/open-api-docs-preview/components/open-api-preview-inputs'
+// Types
+import type { OpenApiDocsViewProps } from 'views/open-api-docs-view/types'
+
+/**
+ * Render an OpenApi docs view alongside preview inputs that allow
+ * dynamic, client-side updating of the props used to render the view.
+ */
+function OpenApiDocsPreviewView() {
+	const [staticProps, setStaticProps] = useState<OpenApiDocsViewProps>()
```

Comment:
I would call this `[apiDocsViewProps, setApiDocsViewProps]` since these are not "static" — they're `fetch`ed from from an API route.

Reply:
Oh nice call! Much better name 💖